L2-Zad.4-AgeVerification/src/main/java/Main/AgeVerification.java

```diff
@@ -0,0 +1,14 @@
+package Main;
+
+public class AgeVerification {
+    public static boolean ageVerification(int age) {
+        if (age >= 18) {
+            return true;
+        } else if(age < 1) {
+            System.out.println("Age must be higher then 0");
+            return false;
+        } else {
+            return false;
```

Comment:
przekombinowane trochę;)

Na razie zapomnijmy o przypadkach, że age < 1 - nie ma sensu tego teraz robić, bo na tym etapie kursu nie obsłuży się tego poprawnie.

Weźmy scenariusz, że wszystko poniżej 18 to niepełnoletni, wszystko równe/powyżej to pelnoletni.

Da się to zrobić.. w jednej linijce :P Podpowiedź: co zwraca wyrażenie "age >= 18"?

Druga uwaga - zamiast 'ageVerification' o wiele praktyczniej byłoby nazwać tę metodę np. 'isAdult', bo wtedy patrz jak czyta się użycie tej metody:

```
if(AgeVerification.ageVerification(23)){ 
... co to znaczy, że wiek został zweryfikowany? Czy ten ktoś ma 13 lat? (niektóre filmy są od tego wieku) 18? (pełnoletność)?
}
```

vs
```
if(AgeVerification.isAdult(23)) {
.... od razu wiadomo, bez wchodzenia w implementacje 'isAdult', że chodzi o pełnoletnośc.
}
```